include/swift/Basic/RelativePointer.h

```diff
@@ -260,6 +260,76 @@ class RelativeIndirectablePointer {
   }
 };
 
+/// A relative reference to an aligned object stored in memory. The reference
+/// may be direct or indirect, and uses the low bit of the (assumed at least
+/// 2-byte-aligned) pointer to differentiate. The remaining low bits store
+/// an additional tiny integer value.
+template<typename ValueTy, typename IntTy, bool Nullable = false,
+         typename Offset = int32_t>
+class RelativeIndirectablePointerIntPair {
```

Comment:
This can go away now, right?

Reply:
(Nevermind, I see it's still used to reserve the unused bit in the protocol reference.)